reduction/line_imaging.py

```diff
@@ -63,9 +68,15 @@
     else:
         exclude_7m = False
 
+if os.getenv('LINE_NAME'):
+    line_name = os.getenv('LINE_NAME').lower()
+else:
+    raise ValueError("line_name was not defined")
+
 # set the 'chanchunks' parameter globally.
 # CASAguides recommend chanchunks=-1, but this resulted in:
-# 2018-09-05 23:16:34     SEVERE  tclean::task_tclean::   Exception from task_tclean : Invalid Gridding/FTM Parameter set : Must have at least 1 chanchunk
+# 2018-09-05 23:16:34     SEVERE  tclean::task_tclean::   Exception from task_tclean :
+#Invalid Gridding/FTM Parameter set : Must have at least 1 chanchunk
```

Comment:
don't change the wrap on comments like this.  Your linter probably suggested this change, but this one needs to be left as is because the formatting matters.